source/index.md

```diff
@@ -81,15 +81,49 @@ Number  Text
 
 Requests can optionally be signed.
 
-The signature is an RSASSA-PKCS1-v1_5 signature of the body represented in Hex.
-The signee must be identified by the signing API-key. Both should be provided
-in a `Signature` header together with `RS256-hex` identifying the signing
-algorithm:
+The signature is an RSA or HMAC signature represented in Hex of the HTTP body.
```

Comment:
I agree. But it's possible to read like "a signature of something; that something is the HTTP body represented in Hex". How one then would represent the HTTP body in Hex ... I dunno.